### PR TITLE
Create FullNames.md

### PR DESCRIPTION
**Pull request by Harshita-Kohli**
Added a new file having full names

### CODE REVIEW

**Comment by Harshita-Kohli** [comment]
added a full name file

File: FullNames.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Here are the full names of our team:
+- Japneet Kaur
```
Comment (AshishD01) [inline FullNames.md]: wrong name , change it @Harshita-Kohli

**Review by AshishD01 — CHANGES_REQUESTED** [review]
reflect all the changess

File: FullNames.md (hunk 1 above)
Comment (Harshita-Kohli) [reply FullNames.md]: ok sure...let me work on it

**Review by AshishD01 — APPROVED** [review]
good

**Pull request merged**